FIPS/fip-common_node_api.md

```diff
@@ -146,17 +146,19 @@ simulataneously, with transport-level correlation agreed out-of-band.
 | ChainGetTipSetByHeight              | State query      | Returns the tipset at the specified height                                                                                                                                                                                                                                                                                                                                                                                                     |
 | ChainHasObj                         | State query      | checks if a given CID exists in the chain blockstore                                                                                                                                                                                                                                                                                                                                                                                           |
 | ChainHead                           | State query      | Returns the chain head (ie. the heaviest tipset)                                                                                                                                                                                                                                                                                                                                                                                               |
 | ChainNotify                         | State query      | Subscribe to changes to the chain head                                                                                                                                                                                                                                                                                                                                                                                                         |
-| ChainPutObj                         |                  | Puts a given object into the block store                                                                                                                                                                                                                                                                                                                                                                                                       |
+| ChainPutObj                         | <uncategorized>  | Puts a given object into the block store                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
This method has been excluded.

Would also be good to assign a category rather than using `<uncategorized>`, for other methods